try-it.md

```diff
@@ -480,36 +474,24 @@ First, you must scale down the deployment of the `baremetal-operator` running
 in the cluster.
 
 ```sh
-kubectl scale deployment capm3-baremetal-operator-controller-manager -n metal3 --replicas=0
+$ kubectl scale deployment baremetal-operator-controller-manager -n baremetal-operator-system --replicas=0
 ```
 
 To be able to run `baremetal-operator` locally, you need to install
 [operator-sdk](https://github.com/operator-framework). After that, you can run
 the `baremetal-operator` including any custom changes.
 
 ```sh
-cd ~/go/src/github.com/metal3-io/baremetal-operator
-make run
+$ cd ~/go/src/github.com/metal3-io/baremetal-operator
+$ make run
 ```
 
 ### 2.5. Running Custom Cluster API Provider Metal3
 
 There are two Cluster API related managers running in the cluster. One
 includes set of generic controllers, and the other includes a custom Machine
-controller for Metal3. If you want to try changes to
-`cluster-api-provider-metal3`, you want to shut down the custom Machine
-controller manager first.
-
-```sh
-$ kubectl scale deployment capm3-controller-manager -n capm3-system --replicas=0
-```
-
-Then you can run the custom Machine controller manager out of your local git tree.
-
-```sh
-cd ~/go/src/github.com/metal3-io/cluster-api-provider-metal3
-make run
-```
+controller for Metal3. You can try changes to
+`cluster-api-provider-metal3` and the changes will be automaticly applied.
```

Comment:
This sounds strange. Is it when using Tilt? In that case I think it should be mentioned. E.g. "... changes will be automatically applied when using Tilt."